release-1.8/release_notes_draft.md

```diff
@@ -278,6 +281,8 @@ kind.
  defaulting is disabled.
  - The default spec.revisionHistoryLimit for all applicable kinds in the
  apps/v1beta2 group version has set to 10.
+ - The default spec.successfulJobsHistoryLimit is 3 and spec.failedJobsHistoryLimit
```

Comment:
The default spec.successfulJobsHistoryLimit is 3 and spec.failedJobsHistoryLimit is 1 on CronJobs

or 

The default spec.successfulJobsHistoryLimit is three and spec.failedJobsHistoryLimit is one on CronJobs

Reply:
Fixed. Thanks for catching that.